qemu/tests/block_copy.py

```diff
@@ -74,8 +73,9 @@ def get_device(self):
         """
         according configuration get target device ID;
         """
-        image_file = storage.get_image_filename(self.parser_test_args(),
-                                                self.data_dir)
+        params = self.parser_test_args()
+        image_name = "%s.%s" % (params["image_name"], params["image_format"])
+        image_file = utils_misc.get_path(self.data_dir, image_name)
```

Comment:
This part also appears in https://github.com/autotest/tp-qemu/pull/1017
You may delete one depends on which patch merges first.

Reply:
@hereischen Yes, just keep here for testing pass, I will delete the redundant part once one of them got merged. Thanks.